docs/contributing/testing-guidelines/index.md

```diff
@@ -1,5 +1,21 @@
 # Testing guidelines
 
-!!! warning
+## Unit testing
 
-    Under Construction
+Unit testing is a software testing method that tests individual units of source code to determine whether they satisfy the specification.
+
+For details, see the [Unit testing guidelines](unit-testing.md).
+
+## Integration tests
+
+Integration Testing combines and tests the individual software modules as a group, and is done after unit testing.
```

Comment:
- Use the same title as https://en.wikipedia.org/wiki/Integration_testing.
- Fix capitalization.
- I think it's not always after unit testing.

```suggestion
## Integration testing

Integration testing combines and tests the individual software modules as a group.
```

Reply:
Ah, but Wikipedia says "It occurs after [unit testing](https://en.wikipedia.org/wiki/Unit_testing) and before [system testing](https://en.wikipedia.org/wiki/System_testing)." :thinking: 

I'll remove this suggestion.